package.json

```diff
@@ -23,7 +24,7 @@
     "branches": 100
   },
   "scripts": {
-    "build": "./node_modules/uglify-es/bin/uglifyjs src.js -cm --toplevel > tinyfunk.js",
+    "build": "./node_modules/uglify-es/bin/uglifyjs src.js -cm --toplevel --wrap tinyfunk > tinyfunk.js",
```

Comment:
Just a tip, `npm` is smart enough to have `node_modules` on its path, so if you'd like, you could reduce this to
```
"build": "uglifyjs src.js -cm --toplevel --wrap tinyfunk > tinyfunk.js",
```

Reply:
Thanks for the review.  I originally included that relative path because at one point I was also building an es5 bundle, and had both `uglify-js` and `uglify-es` as deps.  Sadly, they share the same executable name, so it always called the wrong one.  The relative path fixed that by calling the one I wanted explicitly. 

I removed the es5 build at some point, but forgot to change this command.  Good find.